distros/openshift/security-context-constraints/README.md

```diff
@@ -0,0 +1,9 @@
+# distros/openshift/security-context-constraints
+
+## Security Context Constraints
+
+OpenShift deployments require Security Context Constraints to be applied for containers to function properly. The ones defined here were identified through testing the deployment against an OpenShift cluster and creating SCCs. These are prerequisites, and should be applied before attempting to install Nephio components.
```

Comment:
Do we need to add an entry to the docs for these prerequisites?
https://docs.nephio.org/docs/guides/install-guides/install-on-openshift/

Reply:
I do think documenting this in the docs on a following PR would benefit readers, yes

Reply:
https://github.com/nephio-project/nephio/issues/801